mlflow/gateway/providers/mosaicml.py

```diff
@@ -0,0 +1,134 @@
+from typing import Any, Dict
```

Comment:
Can you please (manually) the scenario described below works as intended?

https://databricks.atlassian.net/browse/MLOPS-167

That is, we are returning an informative error message as opposed to `"too many 500 error responses"`

Reply:
Note: they will start returning 400s from mosaic on this soon, but let's just make sure the current path works.
https://mosaicml.atlassian.net/browse/MCLOUD-2970

Reply:
I think this should be a fast follow to not block this for the 2.7.0 release